### DISCUSSION TITLE
Did not receive confirmation email upon signup

### DISCUSSION THREAD

**Discussion by Ectsang** [Budibase/budibase#11364] [Help] [answered]
I never received an email when I signed up, so now I cannot login. Can someone help reset my account or resend the confirmation email?

**Comment by ConorWebb96** [comment, marked as answer]
Hey @Ectsang,

I believe you sent a request through to the support email. 

I'm just going to write and answer here for others if they stumble across this discussion, please contact `support@budibase.com` and provide relevant details such as; `Email address` used and the `tenant id` used when registering.
